### PR TITLE
fix insert guru names strings

### PR DESCRIPTION
**Pull request by tarunsingh5**
updated to fix:
spelling error in harkrishan
made all har + suffix names one word to be consistent w/ ardaas in BaniDB (see: https://www.sikhitothemax.org/shabad?id=7738&q=pBskg&type=1&source=all&highlight=80058)

### CODE REVIEW

**Review by tsingh777 — COMMENTED** [review]
If we're removing the space should we update the drop down strings too? https://github.com/KhalisFoundation/sttm-desktop/blob/dev/www/locales/en.json#L73

image

**Comment by tarunsingh5** [comment]
sure - ill update

**Comment by tarunsingh5** [comment]
@tsingh777 ptal

File: www/locales/en.json [hunk 1]
```diff
@@ -76,9 +76,9 @@
       "ARJUN_DEV_JI": "Guru Arjun Dev Ji",
       "GOBIND_SINGH_SAHIB_JI": "Guru Gobind Singh Ji",
       "GRANTH_SAHIB_JI": "Guru Granth Sahib Ji",
-      "HAR_GOBIND_SAHIB_JI": "Guru Har Gobind Ji",
-      "HAR_KRISHAN_SAHIB_JI": "Guru Har Krishan Ji",
-      "HAR_RAI_SAHIB_JI": "Guru Har Rai Ji",
+      "HARGOBIND_SAHIB_JI": "Guru Hargobind Ji",
```
Comment (tsingh777) [inline www/locales/en.json:79]: nitpick: the key change was unnecessary.

File: www/locales/en.json [hunk 2]
```diff
@@ -76,9 +76,9 @@
...
+      "HARKRISHAN_SAHIB_JI": "Guru Harkrishan Ji",
+      "HARRAI_SAHIB_JI": "Guru Harrai Ji",
```
Comment (tsingh777) [inline www/locales/en.json:81]: would there be push back? Is there other places where it's spelled this way?

**Review by tsingh777 — COMMENTED** [review]
LGTM. I'd let @maneetpaul merge so release changes can be documented.

File: www/locales/en.json (hunk 2 above)
Comment (tarunsingh5) [reply www/locales/en.json:81]: i think we should be fine here

File: www/locales/en.json (hunk 1 above)
Comment (tarunsingh5) [reply www/locales/en.json:79]: wanted to keep consistency

File: www/locales/en.json (hunk 2 above)
Comment (maneetpaul) [reply www/locales/en.json:81]: I've never seen this spelling before? Is this how it is spelled across other Khalis apps?

Comment (tarunsingh5) [reply www/locales/en.json:81]: i based it on the Gurmukhi from here:

ਪ੍ਰਿਥਮ ਭਗੌਤੀ ਸਿਮਰਿ ਕੈ ਗੁਰ ਨਾਨਕ ਲਈਂ ਧਿਆਇ ॥
ਫਿਰ ਅੰਗਦ ਗੁਰ ਤੇ ਅਮਰਦਾਸੁ ਰਾਮਦਾਸੈ ਹੋਈਂ ਸਹਾਇ ॥
ਅਰਜਨ ਹਰਿਗੋਬਿੰਦ ਨੋ ਸਿਮਰੌ ਸ੍ਰੀ ਹਰਿਰਾਇ ॥
ਸ੍ਰੀ ਹਰਿਕ੍ਰਿਸ਼ਨ ਧਿਆਈਐ ਜਿਸੁ ਡਿਠੇ ਸਭਿ ਦੁਖਿ ਜਾਇ ॥
ਤੇਗ ਬਹਾਦਰ ਸਿਮਰਿਐ ਘਰਿ ਨਉ ਨਿਧਿ ਆਵੈ ਧਾਇ ॥
ਸਭ ਥਾਈਂ ਹੋਇ ਸਹਾਇ ॥੧॥

**Pull request merged**